src/docs/openshift-projects-and-access/common-platform-requests.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Common platform requests
+
+slug: common-platform-requests
+
+description: Describes to users common platform requests 
+
+audience: developer, technical lead
+
+author: Private Cloud
+
+content_owner: Private cloud team
+
+sort_order: 5
+---
+
+# Common platform requests 
+Last updated: **April 15, 2024**
+
+Discover the most commonly searched-for tasks, with links to additional instructions.
+
+## Grant access to an existing private repository in the bcgov-c Github organization
+
+If you’re an administrator for your private repositories, you can add users to your repositories by [managing user access on GitHub](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/managing-an-individuals-access-to-an-organization-repository).
+
+There’s a fee for each user that has access to a private repository in the bcgov-c organization, only grant access to users that need it. The fee for all users is currently covered by the OCIO.
+
+There are a limited number of spots available in the bcgov-c organization. If there are no spots remaining, you’ll receive an error message when you try to add a new user. If this occurs, send a request to [PlatformServicesTeam@gov.bc.ca](mailto:platformservicesteam@gov.bc.ca) for an additional spot to the Platform Services Team. Once we confirm your request has been completed, you can add the new user to your repository.
+
+The Platform Services team removes bcgov-c organization access for users that have not been active for 6 months. Once access has been removed, users must be re-invited to the organization and re-added to a repository to gain access.
+
+## Request a new private repository in the bcgov-c GitHub organization
+
+Only the Platform Services team can create new private repositories in the bcgov-c GitHub organization.
+
+[Submit a request to create a private repository.](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2FBCDevOps%2Fdevops-requests%2Fissues%2Fnew%3Fassignees%3Dcaggles%252C%2BShellyXueHan%252C%2Bmitovskaol%252C%2Bpatricksimonian%26labels%3Dgithub-repo%252C%2Bpending%26template%3Dgithub_repo_request.md%26title%3D)
```

Comment:
```suggestion
[Request for private BCGOV GitHub Repository](https://github.com/BCDevOps/devops-requests/issues/new/choose)
```

Reply:
Linking to general issue creation page, rather than the markdown file for a specific issue.